test/resources/controls.js

```diff
@@ -29,6 +29,17 @@
                     target.setAttribute("aria-hidden", isExpanded);
                     return;
                 }
+                //tooltip
+                var isTooltip = this.getAttribute("aria-hastooltip") === "true";
```

Comment:
I believe this is not used and can be removed. 

### FYI
`aria-hastooltip` is invalid. These attributes are a controlled set with specific meanings. They can't be made up. See https://www.w3.org/TR/wai-aria/states_and_properties#global_states

Reply:
👍 

the right approach would be to use data attribute instead of aria, I suppose.